dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/osgi/HttpConduitConfigApplier.java

```diff
@@ -0,0 +1,388 @@
+/**
```

Comment:
This looks like a change I had made ages ago but later removed. Maybe its still around due to a bad merge? Did we actually change anything that you brought forward from 2.6.2?